src/components/views/settings/index.tsx

```diff
@@ -120,7 +119,7 @@ export const Settings: React.FC<SettingsProps> = ({navigation}) => {
   }
 
   return (
-    <Layouts.Basic heading={t('settings:title')} backgroundColor="#FAFAFA">
+    <Basic heading={t('settings:title')} backgroundColor="#FAFAFA">
```

Comment:
Not really for this PR but wondering why this color is coded inline instead of using a `theme.colors` value.